packages/render-fidelity-tools/test/config.json

```diff
@@ -25,6 +25,20 @@
     }
   ],
   "scenarios": [
+    {
+      "name": "khronos-DragonAttenuation",
```

Comment:
Okay, maybe not the *first* model :) This is a bit complicated (in terms of glTF extensions) and not all the renderers are even showing it properly yet. So, we don't need to showcase it right at the top, but at least in the upper section of pretty models.

Reply:
sorry I missed this